docs/cli-eg.rst

```diff
@@ -98,6 +146,36 @@ you can re-seed the random number generator for each sample with
 ``--sample '( random.seed() ) or random.random() < 0.01'`` or
 similar.
 
+For the USPTO dataset sampling can be achieved by providing a 
+tuple as its argument where the first value is a designator
+string that can be either "path" or "container". This way
+sampling can be done either for compressed files which represents 
+the weekly issued patents (path) or for the patents(container) which
+are located inside the issued file. The second value of the tuple can
+be an USPTO Zip file path or a container respectively.
+
```

Comment:
For the USPTO dataset sampling is performed through a 
provided tuple argument named **data**. The tuple's first value is
a designator string that will be either "path" or "container".
Thus, sampling can be done either on compressed files represented
through the weekly issued patents (path) or on the patents' full text
(container) located within the issued file.
The tuple's second value is a USPTO Zip file path or the patent's text, respectively.